src/finder.js

```diff
@@ -73,29 +70,34 @@ export function findNext (key, node) {
     return null
   }
   const next = node[key] || findByAlias(key, node)
-  return isModule(next) ? loadModule(next) : next
+  return next
 }
 
-export function find (node, args, raw, minimist) {
-  // Accept (node, raw, minimist) as initial arguments
-  if (arguments.length === 3) {
-    const argv = raw(args, optionsByType(findOptions(node)))
-    return find(node, argv._.slice(0), args, raw)
-  }
+const addPaths = (node, path = '.') => {
+  node.path = path
+  forEachObjIndexed((n, key) => {
+    if (isCommand(n) || isNamespace(n)) {
+      addPaths(n, `${path}/${key}`)
```

Comment:
`find` is already recursive so there's no need to visit the whole tree from each node, just adding the path for the current node would suffice.